### DISCUSSION TITLE
有v4到v5的升级指南吗？

### DISCUSSION THREAD

**Discussion by kimmy-wang** [wangeditor-team/wangEditor#4563] [Q&A]
有v4到v5的升级指南吗？

**Comment by wangfupeng1988** [comment]
v5 是重构，你可以直接看文档，看 demo ，其实和 v4 使用上很像。

**Reply by TLovers** [reply]
@wangfupeng1988 王神， 4.7.9 版本的代码clone 下来 需要什么特殊的环境吗。dev跑不起来代码

**Reply by wangfupeng1988** [reply]
没有特殊要求。看这里运行 https://github.com/wangeditor-team/wangEditor/blob/v4.7.13/docs/dev.md#%E8%BF%90%E8%A1%8C%E4%BB%A3%E7%A0%81
